geode-redis/src/main/java/org/apache/geode/redis/internal/ExecutionHandlerContext.java

```diff
@@ -216,6 +217,13 @@ private void executeCommand(ChannelHandlerContext ctx, Command command) throws E
         executeWithoutTransaction(command);
       }
 
+      if (logger.isDebugEnabled() && command.getResponse() != null) {
+        ByteBuf response = command.getResponse()
```

Comment:
Could this copy call and the release call two lines down be moved into getPrintableByteBuf? Or maybe this whole block should be moved to a method named "logResponse(command)". Seems like the "release" call should be in a "finally" block in case an exception is thrown.

Reply:
Fixed.